model/graphql.go

```diff
@@ -0,0 +1,7 @@
+package model
+
+type GraphQLRequest struct {
+	Query         string                 `json:"query"`
+	OperationName string                 `json:"op"`
```

Comment:
Shouldn't this be `operationName`? Confirm with the GraphQL spec

Reply:
Is it compulsory to have first letter in caps ?

Reply:
Im talking about the json part. Let the variable name be as is

Reply:
ok